packages/nova-forms/lib/Form.jsx

```diff
@@ -492,6 +511,9 @@ class Form extends Component {
           onKeyDown={this.formKeyDown}
           disabled={this.state.disabled}
           ref="form"
+          clearField={this.clearField}
+          unclearField={this.unclearField}
```

Comment:
Actually I don't know how exactly system works. I try to learn your by feedbacks.